src/components/ComparisonModal/comparisonModal.js

```diff
@@ -0,0 +1,121 @@
+                    <h3 className='information-container__item-value'>{ pokemon.weight }kg</h3>
+                    <h3 className='information-container__item-name'>Weight</h3>
+                    <h3 className='information-container__item-value'>{ pokemon2.weight }kg</h3>
+                  </div>
+                  <div className='information-container__item'>
+                    <h3 className='information-container__item-value'>{ pokemon.gender }</h3>
+                    <h3 className='information-container__item-name'>Gender</h3>
+                    <h3 className='information-container__item-value'>{ pokemon2.gender }</h3>
+                  </div>
+                </div>
+              </div>
+              <div className='pokemon-container'>
+                <div className='pokemon-container__title'>
+                  <img
+                    className='pokemon-container__title-img'
+                    src={ pokeball }
+                    alt='Pokeball'
+                  />
+                  <h3 className='pokemon-container__title-text'>Abilities</h3>
+                </div>
+                <div className='abilities-container'>
+                  <ul className='abilities-list'>
+                    { pokemon.abilities &&
+                    pokemon.abilities.map(ability => (
+                      <li className='abilities-item'key={ ability.ability.name }>
+                        { ability.ability.name.charAt(0).toUpperCase() + ability.ability.name.slice(1)}
+                      </li>
+                    ))}
+                  </ul>
+                  <ul className='abilities-list'>
+                    { pokemon2.abilities &&
+                    pokemon2.abilities.map(ability => (
+                      <li className='abilities-item right'key={ ability.ability.name }>
+                        { ability.ability.name.charAt(0).toUpperCase() + ability.ability.name.slice(1)}
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              </div>
+              <div className='pokemon-container'>
+                <div className='pokemon-container__title'>
+                  <img
+                    className='pokemon-container__title-img'
+                    src={ pokeball }
+                    alt='Pokeball'
+                  />
+                  <h3 className='pokemon-container__title-text'>Stats</h3>
+                </div>
+                <Chart className='chart' />
+              </div>
+              <h2 className='pokemon-text'>{ pokemon.name.toUpperCase() }</h2>     
+            </div> 
+          </div>  
+        )
+      }
+    </div>
+  );
+}
+
+const mapStateToProps = (state) => {
+  return state
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return{
+    isComparisonModalActive: (state) => dispatch(isComparisonModalActive(state)),
+    isComparing: (state) => dispatch(isComparing(state))
```

Comment:
the prefix 'is' should be used to name boolean values or functions that return boolean, rename your actions to represent verbs (something happened)